packages/api/src/post-build.js

```diff
@@ -96,7 +101,7 @@ class PostBuildHandler {
       if (typeof x.ok !== 'boolean' || !x.id || !x.name) {
         throw new InvalidParameterError('Missing All Test Case Info');
       }
-      const testcase = new TestCaseRun(x.ok ? 'ok' : 'not ok', x.id, x.name);
+      const testcase = new TestCaseRun(x.ok ? 'ok' : 'not ok', x.name);
```

Comment:
You seem to be using `"ok"` and `"not ok"` strings in multiple places -- might be helpful to save them as constants and reuse the constant

Reply:
I think this is actually a quirk of parsing tap files, I don't think I can change it.